Dockerfile

```diff
@@ -1,16 +1,21 @@
-FROM alpine:3.4
+FROM golang:1.11-alpine
 
 RUN apk update && apk upgrade && \
-     apk add ca-certificates && \
+     apk add ca-certificates \
+     xz-dev \
+     musl-dev \
+     gcc && \
      rm -rf /var/cache/apk/*
 
-RUN mkdir /etc/deployments
+RUN mkdir -p /go/src/github.com/mendersoftware/deployments
 
 EXPOSE 8080
 
-COPY ./config.yaml /etc/deployments/
+COPY . /go/src/github.com/mendersoftware/deployments
+RUN cd /go/src/github.com/mendersoftware/deployments && go install
 
-ENTRYPOINT ["/entrypoint.sh", "--config", "/etc/deployments/config.yaml"]
+RUN apk del gcc
+
+ENTRYPOINT ["/entrypoint.sh", "--config", "/go/src/github.com/mendersoftware/deployments/config.yaml"]
 
 COPY ./entrypoint.sh /entrypoint.sh
-COPY ./deployments /usr/bin/
```

Comment:
Does it work without this? I think `go install` installs to `/go/bin`, not `/usr/bin`, so I would expect this step (or a modified one) to still be needed.

Reply:
In the image `GOPATH` is in the `PATH` already, so this is not a problem

Reply:
Besides, this used to copy the static binary into the image, which is no longer how we are doing things if these changes are approved.